wandb/sdk/internal/sender.py

```diff
@@ -716,7 +738,7 @@ def send_output(self, data):
             cur_time = time.time()
             timestamp = datetime.utcfromtimestamp(cur_time).isoformat() + " "
             prev_str = self._partial_output.get(stream, "")
-            line = u"{}{}{}{}".format(prepend, timestamp, prev_str, line)
+            line = "{}{}{}{}".format(prepend, timestamp, prev_str, line)
```

Comment:
why the change?  i think the unicode thing was required for py27

Reply:
oops this might've gotten linted or something, reverting